doc/rest_api/snippet_sharing/use_cases/UC_edit_user.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: edit user
+Scope: User Management
+Level: User Goal
+Actors: Authenticated user, Administrator
+Brief: Users can edit their own account information the same way Administrators can do it for them. Administrators can also change permissions of users.
+
+Scenarios:
+----------------------------
+Precondition: Authenticated, sufficient permissions
+Main success scenario: Changes to the user account have been stored to the database successfully. The operating user is informed about the success through a response from the server.
+Alternative scenario: The submitted inputs could not be validated or the user entry for which changes are requested cannot be found. The server responds with an error message.
+Error scenario: Technical problems prevent the storage process from completing. The operating user is informed about the issue through an error response.
+Postcondition: Changes to the user account are stored in the database successfully and are from now on valid for further requests (e.g. authentication attempts).

```

Comment:
They are valid immediately, although I'm not entirely sure how to properly update the session (token) yet (that only concerns the user who has been changed).